src/AutoRest.CSharp/LowLevel/Output/OperationMethodChainBuilder.cs

```diff
@@ -266,10 +266,6 @@ private void AddUriOrPathParameters(string uriPart, IReadOnlyDictionary<string,
                 {
                     AddParameter(text, requestParameter);
                 }
-                else
```

Comment:
In swagger validator, the parameters defined in URL path are checked against the `parameters` list. So here we can remove the exception safely: https://github.com/Azure/oav/blob/develop/documentation/oav-semantic-errors-reference.md#missing_path_parameter_definition

Reply:
Do we have similar validation in CADL?

Reply:
> Do we have similar validation in CADL?

@AlexanderSher CADL compiler will check the path parameter.
![image](https://user-images.githubusercontent.com/4516/196324963-323dbd39-35dd-4271-83ff-cd1fd898142c.png)